test/reduce_test.cpp

```diff
@@ -317,7 +317,8 @@ struct verify_reduce_with_indices
 
                 // scale the prior dst value and add it to the accumulated value
                 if(!float_equal_zero(beta))
-                    accuVal += convert_type<compType>(output.data[dst_offset] * beta);
+                    accuVal += convert_type<compType>(output.data[dst_offset]) *
```

Comment:
[Q] Do our kernels perform scaling in the tensor data type (`T`) domain as well?

Reply:
The alpha and beta are passed to kernel entrance both as float values,  and them alpha is converted to be same as srcType, beta is converted to be same as dstType for each specific.